static/coffee/flows/controllers.coffee

```diff
@@ -212,7 +212,7 @@ app.controller 'FlowController', [ '$scope', '$rootScope', '$timeout', '$log', '
         showDialog('Invalid Format', 'Audio attachments must be encoded as mp3, m4a, wav, ogg or oga files.')
         return
 
-    if action.type in ['reply', 'send'] and (file.size > 20000000 or (file.name.endsWith('.jpg') and file.size > 500000))
+    if action.type in ['reply', 'send', 'email'] and (file.size > 20000000 or (file.name.endsWith('.jpg') and file.size > 500000))
       showDialog('File Size Exceeded', "The file size should be less than 500kB for images and less than 20MB for audio and video files. Please choose another file and try again.")
       return
 
```

Comment:
Attachments on send email should not allow those type of files, and allow all the rest:
https://github.com/greatnonprofits-nfp/rapidpro/blob/2fed4ba886867143ad80d79da7f3bffd08af6ad1/static/coffee/flows/controllers.coffee#L215

Reply:
Have added type validation and removed validation by size for the email attachment. Or size validation should be?

Reply:
The size I would say that's required under 25MB

Reply:
Have set limit to 25MB for email attachments size.